processfamily/threads.py

```diff
@@ -97,7 +97,10 @@ def forceful_stop_thread(thread):
     if thread.isAlive():
         logger.warning("Stopping thread %s forcefully", thread.getName())
         try:
-            thread._Thread__stop()
+            if sys.version_info.major < 3:
+                thread._Thread__stop()
+            else:
+                thread.stop()
```

Comment:
@dee42 where does `.stop()` come from? It's not documented to be a method of the Thread class in Python 3 (and I looked at the python source code, it's not defined there either).